src/transactions/instances/transactionPayload.ts

```diff
@@ -511,3 +513,187 @@ export class MultiSigTransactionPayload extends Serializable {
     return new MultiSigTransactionPayload(EntryFunction.deserialize(deserializer));
   }
 }
+
+export enum TransactionInnerPayloadVariants {
+  V1 = 0,
+}
+
+/**
+ * Represents any transaction payload that can be submitted to the Aptos chain for execution.
+ *
+ * This is specifically used for orderless transactions.
+ */
+export class TransactionPayloadPayload extends TransactionPayload {
```

Comment:
thoughts? also aligns with the enum name
```suggestion
export class TransactionInnerPayload extends TransactionPayload {
```